core/ibft.go

```diff
@@ -124,13 +126,17 @@ func (i *IBFT) startRoundTimer(ctx context.Context, round uint64) {
 	defer i.wg.Done()
 
 	var (
-		duration     = int(i.baseRoundTimeout)
-		roundFactor  = int(math.Pow(float64(2), float64(round)))
-		roundTimeout = time.Duration(duration * roundFactor)
+		duration        = int(i.baseRoundTimeout)
+		roundFactor     = int(math.Pow(float64(2), float64(round)))
+		roundTimeout    = time.Duration(duration * roundFactor)
+		epochAdditional = 0 * time.Second
```

Comment:
no need to declare this variable in normal block, or no? It might just need to be added in epoch block